### PR TITLE
Exante Brokerage initial setup

### PR DESCRIPTION
**Pull request by alexander-myltsev**
#### Description
Exante initial setup

#### Requires Documentation Change
No documentation changes require

#### How Has This Been Tested?
existing tests pass

#### Types of changes
* [ ]  Bug fix (non-breaking change which fixes an issue)
* [ ]  Refactor (non-breaking change which improves implementation)
* [ ]  Performance (non-breaking change which improves performance. Please add associated performance test and results)
* [x]  New feature (non-breaking change which adds functionality)
* [ ]  Breaking change (fix or feature that would cause existing functionality to change)
* [ ]  Non-functional change (xml comments/documentation/etc)

#### Checklist:
* [x]  My code follows the code style of this project.
* [x]  I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
* [x]  I have added tests to cover my changes.
* [x]  All new and existing tests passed.
* [x]  My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: Common/Brokerages/ExanteBrokerageModel.cs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            {
+                return 1m;
+            }
+
+            if (security.Type == SecurityType.Equity)
+            {
+                return EquityLeverage;
+            }
+
+            throw new ArgumentException($"Invalid security type: {security.Type}", nameof(security));
```
Comment (Martin-Molinero) [inline Common/Brokerages/ExanteBrokerageModel.cs:111]: For Options/Futures we still call `GetLeverage` it's just ignored later by the SetLeverage call see https://github.com/QuantConnect/Lean/blob/master/Common/Securities/BrokerageModelSecurityInitializer.cs#L65, don't think we should throw in these cases but just return 1 instead

File: Common/Orders/Fees/ExanteFeeModel.cs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                case SecurityType.Option:
+                case SecurityType.IndexOption:
+                    var optionsFee = ComputeOptionFee(order);
+                    feeResult = optionsFee.Amount;
+                    feeCurrency = optionsFee.Currency;
+                    break;
+
+                case SecurityType.Future:
+                case SecurityType.FutureOption:
+                    feeResult = 1.5m;
```
Comment (Martin-Molinero) [inline Common/Orders/Fees/ExanteFeeModel.cs:79]: Wouldn't the fee depend on the contract quantity?

**Review by Martin-Molinero — CHANGES_REQUESTED** [review]
Leaving minor request, let's add some basic testing for the brokerage model and the Exante fee model, following the pattern of other brokerages 👍

File: Common/Orders/Fees/ExanteFeeModel.cs (hunk 2 above)
Comment (alexander-myltsev) [reply Common/Orders/Fees/ExanteFeeModel.cs:79]: All Exante can provide I described at:

> Exante's info on fees: https://support.exante.eu/hc/en-us/articles/115005873143-Fees-overview-exchange-imposed-fees?source=search. I think I covered all the parts. How do you propose to expand it?

Do you see any extra info on the subject? I don't.

**Comment by alexander-myltsev** [comment]
@Martin-Molinero Hi Martin! 

Please, check the updates.

**Review by Martin-Molinero — COMMENTED** [review]
Hey! Leaving a couple of minor comments 👍

File: Launcher/config.json [hunk 3]
```diff
@@ -451,6 +460,10 @@
       "transaction-handler": "QuantConnect.Lean.Engine.TransactionHandlers.BrokerageTransactionHandler",
       "history-provider": "BrokerageHistoryProvider"
     },
+    
+    "live-exante": {
+      "live-mode": true
```
[the remark this answers is not in the document]
Comment (alexander-myltsev) [reply Launcher/config.json]: Fixed at 299659272b97adc93ffb8b31c977e28a932e7311

File: Common/Brokerages/BrokerageName.cs [hunk 4]
```diff
@@ -109,6 +109,11 @@ public enum BrokerageName
         /// <summary>
         /// Transaction and submit/execution rules will use ftx us models
         /// </summary>
-        FTXUS
+        FTXUS,
+
+		/// <summary>
```
[the remark this answers is not in the document]
Comment (alexander-myltsev) [reply Common/Brokerages/BrokerageName.cs]: Fixed at 7af630f2a27df7ee692f6ca07145ddb521cdd273

**Review by Martin-Molinero — APPROVED** [review]
Nice!

**Comment by alexander-myltsev** [comment]
@Martin-Molinero Martin, please also merge the commit https://github.com/alexander-myltsev/Lean/commit/c27bb50cc7c11d94bb71966e6172e4a2d8e85623 at the same branch

**Pull request merged**